katz/src/main/kotlin/katz/typeclasses/Applicative.kt

```diff
@@ -21,20 +21,42 @@ interface Applicative<F> : Functor<F>, Typeclass {
 }
 
 data class Tuple2<out A, out B>(val a: A, val b: B) {
```

Comment:
all those TupleX stuff it's ok to be inside Applicative file or it's better move that stuff in another file?